### DISCUSSION TITLE
Live Demo?

### DISCUSSION THREAD

**Discussion by niklasgrewe** [directus/directus#11803] [Archive]
Hi, recently i found this Directus Project: [example.directus.app](example.directus.app). Is this a public demo? If so, what are the login details? Unfortunately I have not found anything in the docs.

**Comment by rijkvanzanten** [comment]
Heya! That's a [Directus Cloud](https://directus.cloud) project instance that we use for the screenshots in the docs. It's not a public beta. That being said, you can sign up for a free trial on Directus Cloud for a quick demo 🙂

**Reply by abdonrd** [reply]
I just tried this, but... I can't login:

<img width="1613" alt="Screenshot 2022-02-22 at 23 48 02" src="https://user-images.githubusercontent.com/1007051/155232630-74c3f752-c2f1-4a5f-a94a-f705fba1a0dc.png">

There is an infinite loop of requests.

**Reply by rijkvanzanten** [reply]
~~yikes, we're on it 👍🏻 ~~ All good now!

**Comment by niklasgrewe** [comment]
@rijkvanzanten thanks for your answer, good to know. When I start with the free Community Cloud can i later upgrade to the Standard Cloud with all my data? Is that possible to start for free and upgrade whatever i want to a higher plan?

**Reply by benhaynes** [reply]
That's the goal! We'll be adding a button to upgrade from Community to Standard very shortly. :)
